wagle-client/.env.development

```
@@ -1,2 +1,4 @@
 // .env.development 공통 백엔드 서버 주소
-NEXT_PUBLIC_BASE_URL=https://wagle-wagle.my-project.cloud/api/v1
\ No newline at end of file
+NEXT_PUBLIC_BASE_URL=https://wagle-wagle.my-project.cloud/api/v1
+//tile api
+NEXT_PUBLIC_STADIA_API_KEY=f4b0370b-ee20-4f93-bc3f-b688c4fc433f
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)